payplug/network.py

```diff
@@ -280,6 +281,9 @@ def _request(self, http_verb, url, data=None, authenticated=True):
         if authenticated:
             headers['Authorization'] = 'Bearer ' + self._secret_key
 
+        if self._api_version:
+            headers['PayPlug-Version'] = '' + self._api_version
```

Comment:
```suggestion
            headers['PayPlug-Version'] = self._api_version
```
Pourquoi ce `'' + ` ?

Je pense qu'il serait bon de modifier `payplug/test/test_network/test_http_client.py ` pour tester le header envoyé (avec éventuellement deux tests : soit le header est présent et bon, soit il est absent).

```python
assert do_request_args[2]['Authorization'] == 'Bearer a_secret_key'
assert do_request_args[2]['PayPlug-Version'] == 'TODO'
```